sentry/src/main/java/io/sentry/IHub.java

```diff
@@ -633,4 +634,7 @@ TransactionContext continueTrace(
    */
   @Nullable
   BaggageHeader getBaggage();
+
+  @Nullable
+  RateLimiter getRateLimiter();
```

Comment:
do we want to make it Internal? or is it good to provide it to the user?

Reply:
good point 👍 I think it doesn't make sense to expose it if it's a final class and not an interface, so I'll mark it as internal for now